README.md

```diff
@@ -1,802 +1,3 @@
-Developing Drools and jBPM
```

Comment:
please revert the README.md file back, we don't want to remove its content or delete it

Reply:
thanks @mareknovotny :heart: